internal/internal_workflow.go

```diff
@@ -623,7 +623,13 @@ func (d *syncWorkflowDefinition) Execute(env WorkflowEnvironment, header *common
 				if result, err := getWorkflowMetadata(rootCtx); err != nil {
 					return nil, err
 				} else {
-					return encodeArg(getDataConverterFromWorkflowContext(rootCtx), result)
+					// Use raw value built from default converter because we don't want to use
+					// user-conversion
+					resultPayload, err := converter.GetDefaultDataConverter().ToPayload(result)
+					if err != nil {
+						return nil, err
+					}
+					return encodeArg(getDataConverterFromWorkflowContext(rootCtx), converter.NewRawValue(resultPayload))
```

Comment:
@cretz do you also have special processing on the client side to match the worker side? Since now the worker will use raw value but right now the client doesn't

Reply:
The only clients for user metadata at this time are UI and CLI, both of which can read this object.

Reply:
Any SDK can make a issue a system query. I am asking how when you did RawValues in other SDKs did you put special processing on the client side in addition to the worker side?

Reply:
Are you asking if we added processing to client side to do special things on `__stack_trace` and `__temporal_workflow_metadata` queries unlike other queries? No we did not. Or are you asking in general when wrote `RawValue` support in SDKs if we did any special processing on _any_ side, then the answer is no, not on client side or worker side, only in the converter (well some SDKs have data type expectations for dynamic things, but not really). So, say in Go, if you have a worker that does not use raw value, you can still `var rawVal RawValue`, `myRun.Get(ctx, &rawVal)` and it will work too.

Reply:
>Are you asking if we added processing to client side to do special things on __stack_trace and __temporal_workflow_metadata queries unlike other queries? 

Yes

>No we did not because we don't expect people to manually call these queries 

Well the CLI will call this query so that would need to support `RawValues` No?

Reply:
The CLI already uses the default payload converter which supports this proto type. We actually do have a homemade raw-value form in CLI for other things, but that's unrelated. Raw value is not a construct of the payload it is a construct of the code, and in this case we only use raw value because they may not support proto types, but CLI does support them.